libs/barista-components/autocomplete/src/autocomplete-trigger.spec.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** The default height of the select items. */
+export const AUTOCOMPLETE_OPTION_HEIGHT = 28;
+
+/** The default max height of the overlay panel containing the options */
+export const AUTOCOMPLETE_PANEL_MAX_HEIGHT = 256;
+
+/**
+ * Calculates the actual height of an option and the maximum panel height
+ * based on a given preferred option height.
+ *
+ * @param preferredOptionHeight The intended option height
+ */
+export function calculateOptionHeight(
+  preferredOptionHeight: number,
+): { height: number; maxPanelHeight: number } {
+  const height =
+    preferredOptionHeight > AUTOCOMPLETE_OPTION_HEIGHT
+      ? preferredOptionHeight
+      : AUTOCOMPLETE_OPTION_HEIGHT;
+
+  return {
+    height,
+    maxPanelHeight:
+      AUTOCOMPLETE_PANEL_MAX_HEIGHT % height === 0
+        ? AUTOCOMPLETE_PANEL_MAX_HEIGHT - height / 2
+        : AUTOCOMPLETE_PANEL_MAX_HEIGHT,
+  };
+}
+
+fdescribe.each([
```

Comment:
please remove the `f` in describe

Reply:
But then I can't use parameterized tests...